lib/rules/declaration-property-value-no-unknown/README.md

```diff
@@ -139,3 +139,145 @@ a { top: --foo(0); }
 ```css
 a { top: --foo(10px); }
 ```
+
+### `atRulesSyntax: { "at-rule": { "prelude": "syntax", "descriptors": { "descriptor": "syntax" } } }`
+
+Extend or alter the at-rules syntax dictionary. This option allows you to define custom at-rules or modify existing ones by specifying their expected syntax. Both `prelude` and `descriptors` are optional. The `prelude` is the part of the at-rule that comes after the at-rule name and before the block (if any). The `descriptors` are like properties specific to the at-rule, and are defined similarly to CSS properties.
+
+[CSS Value Definition Syntax](https://github.com/csstree/csstree/blob/master/docs/definition-syntax.md) is used to define the syntax of at-rules, their preludes, and descriptors. If a definition starts with `|` it is added to the existing definition value if any. See the [CSS syntax reference](https://csstree.github.io/docs/syntax/) for default definitions.
```

Comment:
I refer to the following docs.
https://github.com/csstree/stylelint-validator/blob/master/README.md#atrules